index.js

```diff
@@ -26,6 +26,9 @@ const chooseKey = key =>
 
 const decode = partialRight(jwt.decode, [{ complete: true }])
 
+const stripBearer = token => 
+  token ? token.replace(/^Bearer /, '') : null
```

Comment:
I think we should also strip the lowercase version. Thoughts?

Reply:
don't see any reason not to